lib/mail/message.ex

```diff
@@ -316,4 +325,18 @@ defmodule Mail.Message do
 
     mimetype_fn.(extension)
   end
+
+  defp do_get_attachments(messages, attachments) do
+    List.wrap(messages)
```

Comment:
What is the reason for a collection here rather than iterating over the messages and calling `get_attachments` ?